css/src/components/breadcrumbs.scss

```diff
@@ -0,0 +1,30 @@
+$breadcrumb-font-size: $font-size-8 !default;
+$breadcrumb-spacing-horizontal: 0.25em !default;
+$breadcrumb-spacing-vertical: 0.375em !default;
+$breadcrumb-separator: '\00200B/' !default;
+
+.breadcrumbs {
+	font-size: $breadcrumb-font-size;
+	list-style: none;
```

Comment:
Is this required? I thought we had reset list styles.

Reply:
I don't remember exactly why added it but seems to be working just fine without it. Will update!

Reply:
updated